src/pages/dar_application/TypeOfResearch.js

```diff
@@ -170,19 +170,19 @@ export const TypeOfResearch = hh(class TypeOfResearch extends Component {
             disabled: props.disabled,
           }),
 
-          textarea({
-            style: otherTextStyle,
-            value: props.otherText,
-            onChange: props.otherTextHandler,
-            name: 'otherText',
-            id: 'otherText',
-            maxLength: '512',
-            rows: '2',
-            required: props.other,
-            placeholder: 'Please specify if selected (max. 512 characters)',
-            disabled: props.disabled || !props.other,
-          }),
-
+          props.other ?
+            textarea({
+              style: otherTextStyle,
+              value: props.otherText,
+              onChange: props.otherTextHandler,
+              name: 'otherText',
+              id: 'otherText',
+              maxLength: '512',
+              rows: '2',
+              required: props.other,
+              placeholder: 'Please specify if selected (max. 512 characters)',
+              enabled: true
```

Comment:
Going to add a brief overview of the html syntax we use before adding a code suggestion. The syntax is known as hyperscript, which swaps out JSX or traditional html templating with a more JS functional approach. 

For the example above, you can see that the textarea is rendered with a call to `textarea()`, which is provided by the `react-hyperscript-helpers` package. The argument is a Javascript object that can have HTML/CSS properties like `placeholder`, but there are also target keys that the function will evaluate internally. One such key is `isRendered` whose value will be evaluated as a Boolean. If the value is true, the element will render, otherwise it won't show at all.

Now onto the style suggestion. Instead of using a ternary operator, you can simply add `isRendered: props.other` to the JS object to get the same functionality, but with less typing.

Reply:
Thanks Justin! I will make this change